tests/qemu-tests/tests/test_qemu_system.py

```diff
@@ -13,3 +13,12 @@
 except Exception:
     traceback.print_exc()
     exit(1)
+
+
+try:
+    release_ver = pwndbg.gdblib.kernel.krelease()
+    # release should be int tuple of form: (major, minor, patch)
+    assert len(release_ver) == 3
```

Comment:
Maybe an additional test would be joining the tuple elements together as a string and seeing if that string is contained in `kversion()`? Although I see you're appending `.0` for the patch version in some cases, so you would need to handle that special case.

Reply:
such a test was now added